applications/FluidDynamicsApplication/custom_strategies/strategies/fs_strategy.h

```diff
@@ -642,8 +642,10 @@ class FSStrategy : public SolvingStrategy<TSparseSpace,TDenseSpace,TLinearSolver
 
         double Ratio = NormDv / NormV;
 
-        if ( BaseType::GetEchoLevel() > 0 && rModelPart.GetCommunicator().MyPID() == 0)
-            std::cout << "Fractional velocity relative error: " << Ratio << std::endl;
+        if ( BaseType::GetEchoLevel() > 0 && rModelPart.GetCommunicator().MyPID() == 0){
+            KRATOS_INFO("Fractional Step Strategy : ") << "CONVERGENCE CHECK:" << std::endl;
+            KRATOS_INFO("Fractional Step Strategy : ") << "FRAC VEL.: ratio = " << Ratio <<"; exp.ratio = " << mVelocityTolerance << " abs = " << NormDv << std::endl;        
```

Comment:
I had the following in my mind to change, if its ok with everyone

std::stringstream msg;
msg << std::scientific << "FRAC VEL.: ratio = " << std::setprecision(8) << Ratio << "; exp.ratio = " << mVelocityTolerance << " abs = " << NormDv << std::endl;
KRATOS_INFO("Fractional Step Strategy : ") << msg.str();

Reply:
if the logger is not setting any precision we can ... !

Reply:
I never tried, but I think that the logger should also be able to take and understand the `setprecision` things. If not, it should not be that hard to add support.

Reply:
So shall we add it here and then switch to the logger one once it has the corresponding settings ?

Reply:
I would propose just giving a try to implementing @sunethwarna's approach above (or whatever we want to go with) without the intermediate `stringstream`. If it doesn't work as expected or fails to compile, I can add the required methods to logger in a separate PR.